src/libmongoc/src/mongoc/mongoc-uri.c

```diff
@@ -2197,6 +2265,30 @@ mongoc_uri_get_service (const mongoc_uri_t *uri)
 }
 
 
+const char *
+mongoc_uri_get_service (const mongoc_uri_t *uri)
+{
+   return mongoc_uri_get_srv_hostname (uri);
+}
+
+
+const char *
+mongoc_uri_get_srv_service_name (const mongoc_uri_t *uri)
+{
+   bson_iter_t iter;
+
+   BSON_ASSERT_PARAM (uri);
+
+   if (bson_iter_init_find_case (
+          &iter, &uri->options, MONGOC_URI_SRVSERVICENAME) &&
+       BSON_ITER_HOLDS_UTF8 (&iter)) {
+      return bson_iter_utf8 (&iter, NULL);
+   }
+
+   return "mongodb";
```

Comment:
Should we use a private constant for this?

Reply:
I do not believe it is being used anywhere else, but it would not hurt to give it an explicit name. Will add.

Note: `"mongodb"` also happens to be the default "service name" for [GSSAPI](https://github.com/mongodb/specifications/blob/3a4a6e0f318545c9f050a0000fbd53fc4d9a2790/source/auth/auth.rst#gssapi), but I do not believe these to be relevant enough to justify sharing a constant variable.

Reply:
Ah, I didn't realize `"mongodb"` was also hard-coded in `mongoc_uri_finalize_auth`. I suppose if you want to be consistent with that you can declare the constant within this function instead of statically within the file -- but I won't press the issue.

I originally assumed this would warrant `#define` in the private header, but it looks like that simply isn't done anywhere.